src/installers/install-vi.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Need to call file from: https://github.com/Azure/maximo/blob/main/src/installers/install_manager.sh so install can be tested.

Reply:
We will also need a variable set in the deployment to determine if the user wants to deploy the nvgpu artifacts.

Should be set here:
https://github.com/Azure/maximo/blob/main/src/azure/cloud-init.yaml

and then added as a param for the bicep file.

Reply:
Added VI install to install manager script; added parameters to bootstrap, jumpbox, cloudinit, and install manager.

https://github.com/Azure/maximo/pull/15/commits/fbab56eb95b6fdf6ceb8e6178a7d006fc90140d2